rmarkdowns/Week_3.Rmd

```diff
@@ -85,6 +84,25 @@ Each feature has subcategories, called "tags". The `available_tags()` function l
 available_tags("healthcare")
 ```
 
+```{r}
+# this website is a good reference to find out the available tags https://wiki.openstreetmap.org/wiki/Map_features 
+
+available_tags(feature = "healthcare:speciality")  
```

Comment:
the available_tags function only works for features at the level of the "key". all the healthcare data will fall under the "key" of amenity. the website mentioned in this code chunk has the full list, and there is a category of "healthcare", but no way to just pull the available tags for just "healthcare". definitely a bit of a pain...